app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const { getTopics } = require("./controllers/controllers");
+const app = express();
+
+//GET
+app.get("/api/topics", getTopics);
+
+
+//404 INVALID PATH
+app.get("*/*", (request, response) => {
```

Comment:
Nice, you could consider extracting this into its own file as you add more error handling